astropy/convolution/convolve.py

```diff
@@ -550,7 +550,7 @@ def convolve_fft(array, kernel, boundary='fill', fill_value=0, crop=True,
     if array_size_C > 1*u.GB and not allow_huge:
         raise ValueError("Size Error: Arrays will be {}.  Use "
                          "allow_huge=True to override this exception."
-                         .format(human_file_size(array_size_C.to(u.byte).value)))
+                         .format(human_file_size(array_size_C)))
```

Comment:
Yes, `human_file_size` understands `Quantity`, so no need to convert here. I think this was just missed when `human_file_size` was changed.